scripts/ts/common/slack/postMessage.ts

```diff
@@ -0,0 +1,35 @@
+import fetch from "node-fetch";
+
+const endpoint = "https://slack.com/api/chat.postMessage";
+const slackToken = process.env.IO_APP_SLACK_HELPER_BOT_TOKEN;
+
+/**
+ * Use the Slack API to post a message in a specific channel, using the IO-App helper bot
+ * @param text The message to post
+ * @param channel The target channel
+ * @param unfurlMessage default true, the message will unfurl the links
+ */
+export const slackPostMessage = async (
+  text: string,
+  channel: string,
+  unfurlMessage: boolean = true
+): Promise<unknown> => {
+  const url = new URL(endpoint);
+  const res = await fetch(url, {
+    method: "POST",
+    headers: {
+      Authorization: `Bearer ${slackToken}`,
+      "Content-Type": "application/json; charset=utf-8"
+    },
+    body: JSON.stringify({
+      channel,
+      text,
```

Comment:
@fabriziofff 
I didn't do the code review, but I already meet an issue with slack message API: you have to check the message size because the API could truncate it
https://api.slack.com/changelog/2018-04-truncating-really-long-messages#:~:text=The%20text%20field%20of%20messages,40%2C000%20characters%20in%20August%202018.
I wrote the equivalent in python by using the `block` field
https://github.com/pagopa/pivotal-stories/blob/73811d92583a1d6db4543e9003db8cbb434671fa/src/utils/slack.py#L26

Reply:
Tnx for the alert, in a1fd91ef8a6cc2d980bea8c992136cd31011cfef I've splitted the message in two, in order to separate the fixed static part that need to be in the same message from the dynamic part that could be automatically splitted without problem.